test/fixtures/line/scriptableOptions.js

```diff
@@ -16,8 +16,10 @@ module.exports = {
       },
       plugins: {
         annotation: {
-          drawTime() {
-            return 'afterDatasetsDraw';
+          common: {
+            drawTime() {
+              return 'afterDatasetsDraw';
+            }
```

Comment:
Same as above

Reply:
Yes, took this off from the other PR and forgot to update these.